low-level/sspl-ll.spec

```diff
@@ -94,6 +94,8 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
     ln -sf $SSPL_DIR/low-level/resource_health_view /usr/bin/resource_health_view
     ln -sf $SSPL_DIR/low-level/sspl_bundle_generate /usr/bin/sspl_bundle_generate
     ln -sf $SSPL_DIR/low-level/manifest_support_bundle /usr/bin/manifest_support_bundle
+    ln -sf $SSPL_DIR/low-level/framework $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/health_provider/
```

Comment:
why two soft links here ?

Reply:
This health_provider package is importing framework package like existing generate_resource_health_view.
To avoid `framework module import error`, we need framework symlink under this health_provider package. Otherwise we need to copy this health_provider package and its modules under `$SSPL_BASE/low-level/` directory.

Instead of coping the package and create symlink, this is the simplified way. Thats why two symlink here.
cc: @satish-seagate

Reply:
Both will work without any issue, but I will prefer copying(rather move) to low-level as we are doing same for independently executable scripts.

Reply:
Sure, I will test and commit.